src/java.base/share/classes/jdk/internal/classfile/components/CodeRelabeler.java

```diff
@@ -34,13 +34,14 @@
 import jdk.internal.classfile.instruction.SwitchCase;
 import jdk.internal.classfile.instruction.TableSwitchInstruction;
 import jdk.internal.classfile.Label;
+import jdk.internal.classfile.instruction.CharacterRange;
 import jdk.internal.classfile.instruction.ExceptionCatch;
 import jdk.internal.classfile.instruction.LabelTarget;
 import jdk.internal.classfile.instruction.LocalVariable;
 import jdk.internal.classfile.instruction.LocalVariableType;
 
 /**
- * CodeRelabeler is a {@link jdk.internal.classfile.CodeTransform} replacing all occurences
+ * CodeRelabeler is a {@link jdk.internal.classfile.CodeTransform} replacing all occurrences
```

Comment:
Nit - perhaps just use `A code relabeler is a ...` (instead of using the class name, which then would require another `{@code ... }`.

Reply:
fixed, thanks.